packages/frontend/src/common/components/Table.tsx

```diff
@@ -22,3 +24,15 @@ export function TableHeader({ children }: Props) {
 export function TableBody({ children }: Props) {
   return <tbody>{children}</tbody>
 }
+
+export function TableCell({ children, className }: Props) {
+  return (
+    <td className="p-0 table-fixed break-all">
+      <div className={'p-4 bg-white mb-1 w-full ' + className}>
```

Comment:
What if we use
```suggestion
      <div className=`p-4 bg-white mb-1 w-full ${className}`>
```

Will it work the same way or it has some consequences?

@niall-shaw What do you think about that?

Reply:
We can also do it this way
```
<div className={`p-4 bg-white mb-1 w-full ${className ? className : ''}`}>
```